src/pages/crm-products-page/CrmProductsPage.tsx

```diff
@@ -63,32 +63,30 @@ const CrmProductsPage = () => {
     setStatus(event.target.value)
   }
 
-  const handlePaginationModelChange = useCallback(
-    (paginationModel: GridPaginationModel) => {
-      const categoryId = category === '' ? undefined : parseInt(category)
-      const productStatus =
-        status === '' ? undefined : (status as ProductStatus)
-      if (searchDebounce) {
-        dispatch(
-          searchProductsForCrm({
-            limit: PAGE_SIZE,
-            offset: paginationModel.page * PAGE_SIZE,
-            searchQuery: searchDebounce
-          })
-        )
-      } else {
-        dispatch(
-          fetchProductsForCrm({
-            limit: PAGE_SIZE,
-            offset: paginationModel.page * PAGE_SIZE,
-            productStatus,
-            productCategoryId: categoryId
-          })
-        )
-      }
-    },
-    [status, productsForCrmPageNumber, category, searchDebounce]
-  )
+  const handlePaginationModelChange = (
+    paginationModel: GridPaginationModel
+  ) => {
+    const categoryId = category === '' ? undefined : parseInt(category)
+    const productStatus = status === '' ? undefined : (status as ProductStatus)
+    if (searchDebounce) {
+      dispatch(
+        searchProductsForCrm({
+          limit: PAGE_SIZE,
+          offset: paginationModel.page * PAGE_SIZE,
+          searchQuery: searchDebounce
+        })
+      )
+    } else {
+      dispatch(
+        fetchProductsForCrm({
+          limit: PAGE_SIZE,
+          offset: paginationModel.page * PAGE_SIZE,
+          productStatus,
+          productCategoryId: categoryId
+        })
+      )
+    }
+  }
```

Comment:
> this is not for fix, just a question

why do we have search and fetch as a separate requests? `search-text` should passed in `fetch` as a param, we don't need another separate request for this

Reply:
But we do https://www.test-store.shop/docs#/product/search_all_products_for_crm_api_product_search_for_crm__get

Reply:
okay, thanks, this is a question to back-end side